frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-list/experiment-list.component.html

```diff
@@ -124,7 +124,7 @@
             class="experiment-schedule ft-10-400"
             *ngIf="experiment.state === ExperimentState.SCHEDULED"
           >
-            {{ '(' + (experiment.startOn | date: 'medium') + ')' }}
+            {{ '(' + (experiment.startOn | date: 'MMM d, y h:mm a') + ')' }}
```

Comment:
This change is to address the issue zack mentioned in issue #152 for removing seconds from schedule time as its not required.